packages/beacon-node/src/network/reqresp/ReqRespBeaconNode.ts

```diff
@@ -224,7 +229,7 @@ export class ReqRespBeaconNode extends ReqResp {
   private getProtocolsAtFork(fork: ForkName): [ProtocolNoHandler, ProtocolHandler][] {
     const protocolsAtFork: [ProtocolNoHandler, ProtocolHandler][] = [
       [protocols.Ping(fork, this.config), this.onPing.bind(this)],
-      [protocols.Status(fork, this.config), this.onStatus.bind(this)],
+      [protocols.StatusV2(fork, this.config), this.onStatus.bind(this)],
```

Comment:
may want to move this to after `MetadataV3` to make it easier to maintain why we add StatusV2 by default
or add some more comments for it similar to `MetadataV3`